src/transformers/models/llava/modeling_llava.py

```diff
@@ -277,7 +277,7 @@ def resize_token_embeddings(self, new_num_tokens: Optional[int] = None, pad_to_m
         return model_embeds
 
     def _merge_input_ids_with_image_features(
-        self, image_features, inputs_embeds, input_ids, attention_mask, position_ids
+        self, image_features, inputs_embeds, input_ids, attention_mask, position_ids, labels
```

Comment:
nit: passing `position_ids` as an argument to that method when actually it is never used inside that method but defined at the end is weird but since it wasn't really the purpose of that PR, I left it as it is. let me know if you want me to fix that

Reply:
Alright to remove it, should not be given as an input to the model IMO

Reply:
done